.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+    - "0.12"
```

Comment:
Any reason for leaving 0.12? (Not feeling too strong about this, just being curious)